pkgs/development/interpreters/micropython/default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv, lib, fetchFromGitHub, pkgconfig, libffi, python3, readline }:
+
+stdenv.mkDerivation rec {
+  pname = "micropython";
+  version = "1.11";
+
+  src = fetchFromGitHub {
+    owner  = "micropython";
+    repo   = "micropython";
+    rev    = "v${version}";
+    sha256 = "16pj6xpiai4y4g8fhn3b824zibgfv62l4ib091pj930q07g109bv";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [ pkgconfig python3 ];
+
+  buildInputs = [ libffi readline ];
+
+  doCheck = true;
+
+  buildPhase = ''
```

Comment:
how about using `makeFlags = [ "-C mpy-cross" "-C ports/unix"];` instead of a custom `buildPhase`?

Reply:
`makeFlags = [ "-C mpy-cross" "-C ../ports/unix"];` did not work, so changed to `make -C mpy-cross; make -C ports/unix`